### ISSUE TITLE
Syntheses two faces through Style-GAN

### ISSUE THREAD

**Issue by faresbougourzi** [rosinality/stylegan2-pytorch#121]
I am wondering if I can use Style-GAN to synthesis two faces as shown in style-GAN official website

https://user-images.githubusercontent.com/18519110/93667991-95d96900-fa81-11ea-994c-83b4be9d857a.png

Is there any implementation for that in Pytorch

**Comment by rosinality** [comment]
Yes, you only need to input two noises to the generator, like this:

```python
sample_z1 = torch.randn(10, 512, device='cuda')
sample_z2 = torch.randn(10, 512, device='cuda')

sample, _ = g_ema([sample_z1, sample_z2])
```

**Comment by faresbougourzi** [comment]
Thanks for your quick reply, 
What I should do, if I have two images and I want to synthesis them, instead of two random noises 
Best regards

**Comment by rosinality** [comment]
You need to project images to latent spaces and then you can mix them. How to project image is open question, but you can try to use projector.py.

**Comment by faresbougourzi** [comment]
Thanks a lot
